plugins/carla/cmake/lilv.cmake

```diff
@@ -0,0 +1,91 @@
+# base config
+if(MSVC)
+  set(carla_lilv_compile_options /wd4005 /wd4090 /wd4133 /wd4244 /wd4267 /wd4273)
+elseif(CLANG)
+  set(carla_lilv_compile_options -Wno-error -Wno-deprecated-declarations -Wno-implicit-fallthrough
+                                 -Wno-incompatible-pointer-types-discards-qualifiers -Wno-unused-parameter)
+else()
+  set(carla_lilv_compile_options
+      -Wno-error
+      -Wno-deprecated-declarations
+      -Wno-discarded-qualifiers
+      -Wno-format-overflow
+      -Wno-implicit-fallthrough
+      -Wno-maybe-uninitialized
+      -Wno-unused-parameter)
+endif()
+
+set(carla_lilv_basedir carla/source/modules/lilv)
+
+set(carla_lilv_include_directories carla/source/includes ${carla_lilv_basedir}/config)
+
+if(NOT
+   (OS_FREEBSD
+    OR OS_MACOS
+    OR OS_WINDOWS))
+  set(carla_lilv_extra_libs dl m rt)
+endif()
+
+# serd
+add_library(carla-lilv_serd STATIC)
+
+set_target_properties(carla-lilv_serd PROPERTIES OSX_ARCHITECTURES "x86_64;arm64" POSITION_INDEPENDENT_CODE ON)
+
+target_compile_options(carla-lilv_serd PRIVATE ${carla_lilv_compile_options})
+
+target_include_directories(carla-lilv_serd PRIVATE ${carla_lilv_include_directories} ${carla_lilv_basedir}/serd-0.24.0)
+
+target_sources(carla-lilv_serd PRIVATE ${carla_lilv_basedir}/serd.c)
+
+# sord
+add_library(carla-lilv_sord STATIC)
+
+set_target_properties(carla-lilv_sord PROPERTIES OSX_ARCHITECTURES "x86_64;arm64" POSITION_INDEPENDENT_CODE ON)
+
+target_compile_options(
+  carla-lilv_sord
+  PRIVATE ${carla_lilv_compile_options}
+          # workaround compiler bug, see https://gcc.gnu.org/bugzilla/show_bug.cgi?id=109585
+          $<$<NOT:$<BOOL:${MSVC}>>:-fno-strict-aliasing>)
```

Comment:
We disable strict aliasing by default, so this shouldn't be necessary.

We would welcome new code that actually _works_ with strict aliasing, but that's not a requirement.